dogs.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<!--HEAD-->
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>
+      Adopt a Dog
+    </title>
+    <!---<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">-->
+
+    <!--CSS-->
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+  </head>
+
+  <!--BODY-->
+  <body>
+    <a href="#main-content" class="hidden">Skip to main content</a>
+    <header>
+      <div class="head-bucket">
+        <div class="head-1">
+          <img src="images/adoptadog-logo.svg" class="logo" alt="white logo of a dog">
+          <h2 class="headline"><em>Seattle's premier dog adoption agency</em></h2>
+        </div>
+
+        <!--CHECKOUT-->
+        <div class="head-2">
+          <img src="images/cart-icon.svg" class="cart" alt="icon of a white shopping cart" width="15">
+          <a href="checkout.html" class="button" target="_blank">Checkout</a><!--need to create CSS for buttons-->
+        </div>
+      </div><!--close head-bucket div-->
+    </header>
+
+      <!--NAVIGATION-->
+      <ul>
+        <li><a href="index.html" class="a-navigation" target="_blank">About</a></li>
+        <li><a href="dogs.html" class="a-navigation active" target="_blank">Dogs</a></li>
+        <li><a href="blog.html" class="a-navigation" target="_blank">Blog</a></li>
+        <li><a href="contact.html" class="a-navigation" target="_blank">Contact</a></li>
+      </ul>
+
+    <!--MAIN CONTENT-->
+    <h2>Here are all our dogs!</h2>
+    <div class="grid-container">
+      <div class="grid-itb-1">
```

Comment:
You'll want to use the same class for all of the dog tiles. And I'd encourage you to use more semantic class names